website/project/views/register.py

```diff
@@ -40,25 +38,18 @@
 from website.identifiers.client import EzidClient
 
 from .node import _view_project
-from .. import clean_template_name
-
 
 @must_be_valid_project
 @must_have_permission(ADMIN)
-@must_not_be_registration
 def node_register_page(auth, node, **kwargs):
 
-    ret = {
-        'options': [
-            {
-                'template_name': metaschema['name'],
-                'template_name_clean': clean_template_name(metaschema['name'])
-            }
-            for metaschema in OSF_META_SCHEMAS
-        ]
-    }
-    ret.update(_view_project(node, auth, primary=True))
-    return ret
+    if node.is_registration:
```

Comment:
A docstring for this view function wouldn't go amiss. (and there's already a blank line just waiting to hold it!)